.github/workflows/codeql.yml

```diff
@@ -0,0 +1,39 @@
+name: "Static Analysis: CodeQL Scan"
+
+on:
+  push:
+    branches:
+      - "pull-request/[0-9]+"
+      - "main"
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event_name }}
+  cancel-in-progress: true
+
+jobs:
+  analyze:
+    name: Analyze (${{ matrix.language }})
+    runs-on: ubuntu-latest
+    permissions:
+      security-events: write
+
+    strategy:
+      fail-fast: false
```

Comment:
Should we do `fail-fast: false` for the bandit scan as well?

Reply:
There is no matrix to fail fast on is there?

Reply:
🤦 I misinterpreted the docs to be all workflows instead of job matrix within a workflow. Resolving this.